third_party/freertos/wscript

```diff
@@ -13,6 +13,9 @@ def build(bld):
     elif bld.env.MICRO_FAMILY == 'NRF52840':
         freertos_port_name = 'ARM_CM4F'
         use_timers = True
+    elif bld.env.MICRO_FAMILY == 'SF32LB':
+        # FIXME: not true, but it allows to boot for now
+        freertos_port_name = 'ARM_CM4F'
```

Comment:
probably this will not work for a real boot into userland.  let me add cm33 that I did for nRF53 into FreeRTOS